src/components/Carousel/PrevNext.js

```diff
@@ -13,6 +13,7 @@ function PrevNext({ onClick, type }) {
         position: absolute;
         z-index: 1;
         top: 50%;
+        height: ${6 * GU}px;
```

Comment:
Good idea yes :+1: 

Should we also set the width to 6gu?

Reply:
As discussed offline, will keep it at the current width to keep the icon small and revisit the sizing later.